src/client/src/orangehrmAdminPlugin/pages/languageTranslation/languageTranslation.vue

```diff
@@ -0,0 +1,285 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <oxd-table-filter :filter-title="$t('admin.translate_language_package')">
+      <oxd-form @submitValid="onSubmit" @reset="onReset">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                :value="languagePackage"
+                :label="$t('admin.language_package')"
+                disabled
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                :value="sourceLanguage"
+                :label="$t('admin.source_language')"
+                disabled
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <language-group-list-dropdown v-model="filters.groupId" />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                v-model="filters.sourceText"
+                :label="$t('admin.source_text')"
+                :rules="commentValidators"
```

Comment:
better use a rule object with field name here. `rules.sourceText`

Reply:
remove the length validation because Kavinda mentions do not need it